content/intro-to-storybook/angular/en/screen.md

```diff
@@ -321,9 +324,75 @@ Cycling through states in Storybook makes it easy to test we’ve done this corr
   />
 </video>
 
+## Interactive stories
+
+So far, we've been able to build a fully functional application from the ground up, starting from a simple component up to a screen, continuously testing each change using our stories. But with each story added, we would also need to check each variation manually to ensure the UI doesn't break, which requires extra work.
+
+Couldn't we automate this workflow and interact with our components automatically?
+
+Storybook's [`play`](https://storybook.js.org/docs/6.4/angular/writing-stories/play-function) function allows us to do just that. They are small snippets of code that run after the story renders. Ideal for scenarios such as ours where we would like to see what happens to our UI when we update our tasks. Since they rely on using framework-agnostic DOM APIs, we can write stories with the play function to interact with the UI and automate human behavior no matter the framework we use.
```

Comment:
If these are changes _for_ `6.4`, I don't think the version should be in the URL, as the correct reference would be "latest" when these are published.

Reply:
This is provisional so that someone doesn't ask why is it pointing to a 404.